File: TrainingRecommender/Models/Training.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TrainingRecommender.Models
{
    public class Training
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public int Duration { get; set; } // днів
        public double TrainingRate { get; set; } // днів на тиждень
        public string About { get; set; }
        public int Score { get; set; }
        public Level Level { get; set; }
        public IEnumerable<Exercise> Exercises { get; set; }
        public IEnumerable<TrainingMuscle> Muscles { get; set; }
        public IEnumerable<UserTraining> UserTrainings { get; set; }
    }

    public enum Level
    {
        Easy,
        Medium,
        Hard
    }
}

File: TrainingRecommender/ClientApp/src/app/models/training.ts
import { Exercise } from './exercise';
import { TrainingMuscle } from './training-muscle';
import { UserTraining } from './user-training';

export class Training {
  id: number;
  title: string;
  duration: number;
  trainingRate: number; // днів на тиждень
  about: string;
  level: Level;
  score: number;
  exercises: Exercise[] = [];
  muscles: TrainingMuscle[];
  isRecommended = false;
  userTrainings: UserTraining[];
}

enum Level {
  Easy,
  Medium,
  Hard
}
